lib/jormungandr/test/unit/Cardano/Wallet/Jormungandr/NetworkSpec.hs

```diff
@@ -580,7 +597,8 @@ instance Arbitrary S where
             [ (50, pure 0)
             , (50, choose (0, 2 * fromIntegral k))
             ]
-        targetChainLength <- (+ initialChainLength) <$> choose (0, 2 * fromIntegral k)
+        targetChainLength <-
+            (+ initialChainLength) <$> choose (0, 2 * fromIntegral k)
         bootstrap <- genNodeOps mockNodeK initialChainLength emptyNode
         let node = applyNodeOps (mconcat bootstrap) emptyNode
         operations <- genNodeOps mockNodeK targetChainLength node
```

Comment:
I wonder what was the motivation to go over and modify this module which isn't really related to this PR? 
Generally speaking, this kind of changes adds up in a PR and obfuscate a bit the real subject. I am all for refactoring code and updating code but it should always serve the ongoing task: there's no point refactoring a piece of code that is working and that doesn't need new features or bug fixes (a good read on the topic would be: https://martinfowler.com/books/refactoring.html). 

If this kind of stuff is really itchy, I'd recommend separating these in another PR to isolate the rest of the PR from it :+1:

Reply:
> I wonder what was the motivation to go over and modify this module which isn't really related to this PR?

The reason is very simple:

1. While working on the `Jormungandr.NetworkSpec` module, I noticed that some of the code didn't follow our team coding standard.
2. It was easy to fix, so I fixed it there and then, placing the changes into a separate commit.

> If this kind of stuff is really itchy, I'd recommend separating these in another PR to isolate the rest of the PR from it +1

It would also be my preference to create a separate PR for such changes.

However, when I did that before, you expressed a preference of placing such changes in a _separate commit_ of the PR to which it was related, rather than a separate PR:

https://github.com/input-output-hk/cardano-wallet/pull/117#issuecomment-476240560

Ever since then, I've placed such changes into a _separate commit_, rather than a separate PR. Of course, if you'd prefer that I revert back to putting such changes in a _separate PR_, I'd be happy to do that. Which method would you prefer?

> (a good read on the topic would be: https://martinfowler.com/books/refactoring.html).

Thanks for the recommendation, I read this book many years ago. It's a good book, and I agree with many of the ideas in it. :+1:

Reply:
@jonathanknowles I am glad you've dug that comment, and glad to see that I am consistent over almost a year about this :sweat_smile: ... The main point of my comment in #117 was actually even in **bold** and it was also the main point of my question here. 

Now looking back at the commit history, I see the diff:

```diff
-    { getTipId = do
+    { getAccountState = \_ -> error "mock getAccountState"
+
+    , getTipId = do
```

I do think it's a bit excessive regarding coding standard and alignment. I mean, this sole module has `17 insertions(+), 37 deletions(-)`, for what is actually a `3 insertions(+), 1 deletion(-)`. And the PR feels also a bit cluttered through code style changes and at this point, it has a negative impact of the "reviewability" of the PR. 

My point is that, these style changes are not helping much the addition (although I understand that a) one can be tempted to adjust lines as one runs into them, b) it may improve one ability to read through the module). 

The right middle-ground IMO is to limit the code style changes to the functions you are actually touching / changing (so, more restrictive than a _module_). This way, there's no need to load the surrounding in mind when reviewing and, we still get to tidy up the codebase steadily.

Reply:
> The right middle-ground IMO is to limit the code style changes to the functions you are actually touching / changing (so, more restrictive than a module).

Fair enough. I've revised the change a bit so that code style changes are restricted to just the functions that are touched by this PR. Let me know if you need it to be reduced further.

**Regarding our PR process in general**:

Perhaps it would be a good idea for us to document our policy on code style fixes, so we have something that we can refer to?

How about having something like the following guidelines:

> 1. A normal PR _may_ include fixes to code style, but any such fixes should be in a _**single, separate commit**_, and reformatting should be limited to **_only_** the top-level functions that are changed by the PR.

> 2. For _**larger**_ code style fixes that go beyond what's allowed within a normal PR, these should be placed in a _**dedicated**_ PR that introduces no other changes. Such PRs should only be reviewed and merged during _**recovery weeks**_.

Thoughts?

Reply:
@jonathan.knowles I am happy to introduce these additions to the top of the style guide, if others are also happy with it :+1: